src/platform/datapath_raw_xdp_win.c

```diff
@@ -165,32 +165,37 @@ CxPlatGetRssQueueProcessors(
     _Out_writes_to_(*Count, *Count) uint32_t* Queues
     )
 {
+    UNREFERENCED_PARAMETER(Xdp);
```

Comment:
Unless we expect to need this in the future, let's remove it from being passed in.